### PR TITLE
Fix DS108 in cases of descriptions ending with a dot inside

### PR DESCRIPTION
**Pull request by YuvHayun**
## Description
Fixed an issue where **validate** falsely failed with error `DS108` on descriptions ending with brackets with a dot inside.

### CODE REVIEW

File: demisto_sdk/commands/common/tools.py [hunk 1]
```diff
@@ -3934,3 +3934,18 @@ def parse_int_or_default(value: Any, default: int) -> int:
         return int(value)
     except (ValueError, TypeError):
         return default
+
+
+def is_sentence_ends_with_bracket(description):
```
Comment (GuyAfik) [inline demisto_sdk/commands/common/tools.py:3939]:
```suggestion
def is_sentence_ends_with_bracket(description: str):
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Changelog
 ## Unreleased
+* Fixed an issue where **validate** falsely failed with error `DS108` on descriptions ending with brackets with a dot inside.
```
Comment (GuyAfik) [inline CHANGELOG.md:3]:
```suggestion
* Fixed an issue where **validate** falsely failed with error `DS108` on descriptions ending with brackets that contains a dot at the end of them.
```

**Review by GuyAfik — APPROVED** [review]
@YuvHayun do we consider this case valid?

Shouldn't the dot be at the end anyway?

**Comment by YuvHayun** [comment]
> @YuvHayun do we consider this case valid?
> 
> Shouldn't the dot be at the end anyway?

No, as per @ShirleyDenkberg it can be either inside or outside the brackets.

**Pull request merged**